### PR TITLE
Add support for the MIT license.

### PR DESCRIPTION
**Pull request by petermattis**
Add more locations to look for licenses.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/3780)

### CODE REVIEW

File: build/deplicense.sh [hunk 1]
```diff
@@ -69,6 +81,9 @@ function inspect() {
       elif isLGPLV3 "${file}"; then
         echo "GNU Lesser General Public License 3.0"
         return
+      elif isMIT "${file}"; then
+	echo "MIT License"
```
Comment (tschottdorf) [inline build/deplicense.sh]: indentation

**Comment by tschottdorf** [comment]
LGTM

File: build/deplicense.sh (hunk 1 above)
Comment (petermattis) [inline build/deplicense.sh]: Done.

**Comment by petermattis** [comment]
I tweaked the way we find potential files to look for again so that we use more shell expansion. PTAL.

File: build/deplicense.sh [hunk 2]
```diff
@@ -43,13 +43,19 @@ function isLGPLV3() {
...
+}
+
 function inspect() {
   local dir="$1"
 
-  local files="${toplevel}/LICENSE"
-  files="${files} ${toplevel}/COPYING"
-  files="${files} ${toplevel}/internal/LICENSE"
-  files="${files} ${toplevel}/internal/COPYING"
+  local files="$(ls ${toplevel}/{LICENSE,COPYING} 2>/dev/null)"
```
Comment (tschottdorf) [inline build/deplicense.sh]: My bash hygiene spidey senses tingle at the sight of `ls` in shell scripting. This looks like it should be some invocation of `find "${toplevel}" -type f -iname '....' -o <...>`

Comment (tschottdorf) [inline build/deplicense.sh]: whitespace handling is generally broken in this script. Should we fix it?

Comment (petermattis) [inline build/deplicense.sh]: I'd say fix it when it becomes a problem. This script is only run once in a blue moon to satisfy the lawyers.

**Comment by tschottdorf** [comment]
ok, LGTM

**Pull request merged**